app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CreateEventActivity.kt

```diff
@@ -313,54 +290,42 @@ class CreateEventActivity : ActivityForAccessingTumOnline<CreateEventResponse>(R
         }
         binding.eventEndDateView.setOnClickListener {
             hideKeyboard()
-            ThemedDatePickerDialog(this, { _, year, month, dayOfMonth ->
+            DatePickerDialog(this, { _, year, month, dayOfMonth ->
                 end = end.withDate(year, month + 1, dayOfMonth)
                 updateDateViews()
             }, start.year, start.monthOfYear - 1, start.dayOfMonth).show()
         }
 
         // TIME
-        binding.eventStartTimeView.setOnClickListener {
+        binding.eventStartTimeView.setOnClickListener { _ ->
             hideKeyboard()
-            ThemedTimePickerDialog(
-                this,
-                { timePicker, hour, minute ->
-                    timePicker.layoutParams = LinearLayout.LayoutParams(
-                        LinearLayout.LayoutParams.WRAP_CONTENT,
-                        LinearLayout.LayoutParams.WRAP_CONTENT
-                    )
-                    val eventLength = end.millis - start.millis
-                    start = start.withHourOfDay(hour).withMinuteOfHour(minute)
-                    end = end.withMillis(start.millis + eventLength)
-                    updateTimeViews()
-                },
-                start.hourOfDay,
-                start.minuteOfHour
-            ).show()
-        }
-
-        binding.eventEndTimeView.setOnClickListener {
+            TimePickerDialog(this, { _, hour, minute ->
+                val eventLength = end.millis - start.millis
+                start = start.withHourOfDay(hour).withMinuteOfHour(minute)
+                end = end.withMillis(start.millis + eventLength)
+                updateTimeViews()
+            }, start.hourOfDay, start.minuteOfHour, true).show()
+        }
+
+        binding.eventEndTimeView.setOnClickListener { _ ->
             hideKeyboard()
-            ThemedTimePickerDialog(this, { _, hour, minute ->
-                end = end.withHourOfDay(hour)
-                    .withMinuteOfHour(minute)
+            TimePickerDialog(this, { _, hour, minute ->
```

Comment:
Does this look good in the dark+light mode? (afaik that was the reason why ThemedTimePickerDialog was introduced)